bindings/gumjs/gumdukvalue.c

```diff
@@ -973,6 +973,40 @@ _gum_duk_require_pointer (duk_context * ctx,
 }
 
 gboolean
+_gum_duk_is_uint64 (duk_context *ctx,
+                    duk_idx_t index,
+                    GumDukCore *core)
```

Comment:
This one should be moved right above `_gum_duk_get_uint64`, made `static`, and not have an underscore prefix (since it's `static`).